client/src/js/services/PeriodService.js

```diff
@@ -0,0 +1,14 @@
+angular.module('bhima.services')
+.service('PeriodService', PeriodService);
+
+PeriodService.$inject = ['PrototypeApiService'];
+
+function PeriodService(PrototypeApiService) {
+  var service = this;
+
+  // inherit methods from the
+  angular.extend(service, PrototypeApiService);
```

Comment:
There is no `service.url` provided here - does this know where to make requests to?